test/integration/local/test_dummy_multi_model.py

```diff
@@ -38,8 +38,8 @@ def container():
         proc = subprocess.Popen(command.split(), stdout=sys.stdout, stderr=subprocess.STDOUT)
 
         attempts = 0
-        while attempts < 5:
-            time.sleep(3)
+        while attempts < 10:
+            time.sleep(60)
```

Comment:
Note: some tests require that the wait time be as long as 'retry' for the model_server.py to retrieve the mms processes.

Reply:
Do we have to sleep for 60s before the first attempt? I think that means this fixture will never take less than one minute to start up, where it took 3s to run before in happy case. Perhaps we should first sleep for a small amount of time, then 10m in loop after that?